plugin.video.roosterteeth/resources/lib/roosterteeth_list_shows.py

```diff
@@ -88,21 +104,62 @@ def getVideos(self):
                 ADDON, VERSION, DATE, "len(shows)", str(len(shows))), xbmc.LOGNOTICE)
 
         for show in shows:
+            video_list_page_url_http = str(self.video_list_page_url).replace("https", "http")
+            video_list_page_url_https = str(self.video_list_page_url).replace("http", "https")
+
             # skip show if it doesn't contain the website show url (https or http)
-            if str(show.a).find(self.video_list_page_url) < 0:
-                video_list_page_url_http = str(self.video_list_page_url).replace("https", "http")
-                if str(show.a).find(video_list_page_url_http) < 0:
-                    if self.DEBUG == 'true':
-                        xbmc.log("[ADDON] %s v%s (%s) debug mode, %s = %s" % (
-                            ADDON, VERSION, DATE, "skipped show that doesn't contain website show url",
-                            str(self.video_list_page_url)), xbmc.LOGNOTICE)
-                        xbmc.log("[ADDON] %s v%s (%s) debug mode, %s = %s" % (
-                            ADDON, VERSION, DATE, "skipped show that doesn't contain website show url",
-                            str(video_list_page_url_http)), xbmc.LOGNOTICE)
-                        xbmc.log("[ADDON] %s v%s (%s) debug mode, %s = %s" % (
-                            ADDON, VERSION, DATE, "skipped show that doesn't contain website show url",
-                            str(show)), xbmc.LOGNOTICE)
-                    continue
+            if str(video_list_page_url_http).find(RECENTLYADDEDURL) < 0:
+                if str(video_list_page_url_https).find(RECENTLYADDEDURL) < 0:
```

Comment:
Why do you check the https ones? the consts (RECENTLYADDEDURL, ...) all are http://...
you can also get rid of all the str()